src/Microsoft.PowerShell.ConsoleHost/host/msh/CommandLineParameterParser.cs

```diff
@@ -18,6 +18,8 @@
 
```

Comment:
@iSazonov, your last commit had 1 failures in `PowerShell-CI-macos`
ConsoleHost unit tests.Redirected standard input for 'interactive' use.Redirected input explicit prompting (-File -)
```powershell
Expected strings to be the same, but they were different.
Expected length: 7
Actual length:   57
Strings differ at index 0.
Expected: 'PS> 1+1'
But was:  'Copyright ...'
at <ScriptBlock>, /Users/runner/runners/2.163.1/work/1/s/test/powershell/Host/ConsoleHost.Tests.ps1: line 533
533:             $process.StandardOutput.ReadLine() | Should -Be "PS> 1+1"
```